src/index.js

```diff
@@ -3,11 +3,16 @@ import sortAny from 'sort-any';
 
 const sortDeep = (object) => {
   if (!Array.isArray(object)) {
-    if (!(typeof object === 'object') || object === null) {
+    // == null checks for undefined as well as null.
+    if (object == null || (typeof object !== 'object')) {
```

Comment:
for `undefined`, `typeof object === 'object'` is false

Reply:
Quite right. Didn't see that then. Would you like me to revert this line?

Reply:
no, thanks, I'm working on fixing this issues, with prodiving some tests